documentation/asciidoc/topics/con_anti_affinity.adoc

```diff
@@ -0,0 +1,31 @@
+[id='anti_affinity-{context}']
+= Anti-Affinity Strategies
+Each {brandname} node in a cluster is a pod that runs on an {k8s} node in a cluster.
+Each {openshift} node runs on a physical host system.
+Anti-affinity works by distributing {brandname} nodes across {k8s} nodes, ensuring that your {brandname} clusters remain available even if hardware failures occur.
+
+[discrete]
+== Strategies
+
+{ispn_operator} offers two anti-affinity strategies:
+
+Hostname:: {brandname} replica pods are scheduled on different {k8s} nodes.
+Zone:: {brandname} replica pods are scheduled across multiple zones.
+
+[discrete]
+== Fault tolerance
+
+`spec.replicas`:: Defines the number of nodes in each {brandname} cluster.
+`num_owners`:: Is the cache configuration attribute that defines the number of replicas for each entry in the cache.
+
+.Hostname strategy
+Provides tolerance of `x` node failures for the following types of cache:
+
+* Replicated: `x = spec.replicas - (spec.replica - 1)` where `spec.replicas >= num_owners`
+* Distributed: `x = num_owners - 1`
+
+.Zone strategy
+Provides tolerance of `x` zone failures when `x` zones exist for the following types of cache:
+
+* Replicated: `x = spec.replicas`
+* Distributed: `x = num_owners`
```

Comment:
This is still `num_owners - 1`. If there's 3 zones and num_owners=2, it's not always possible to tolerate 2 zone failures as a cache entry could have been exclusively stored on the 2 zones that died.

Reply:
fixed.